tests/ro/creator/data/roc-02-emobon-observatory.yml

```
@@ -0,0 +1,70 @@
+uses: !strategy emobon-observatory  # directive to drive specific 
+base:  !resolve "{domain}/{repo}"
+name: "observatory-bpns-crate"
+datePublished: !resolve "{now}"
+description: "The EMO BON sampling data from the observatory BPNS",
+spatialCoverage:
+  at: [51.44333331, 2.808331]
+  in: "mrgid:26567"
+temporalCoverage:  # same for all obs?
+  start: 2021-07-01
+  end: !resolve "{today}"
+license: https://creativecommons.org/licenses/by/4.0/ # same for all obs?
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)